CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [#1591](https://github.com/bbatsov/projectile/issues/1591): Add `project.el` integration that will make Projectile the default provider for project lookup.
 * Add new command `projectile-find-references` (bound to `C-c C-p ?` and `C-c C-p s x`).
 * [#1737](https://github.com/bbatsov/projectile/pull/1737): Add helpers for `dir-local-variables` for 3rd party use. Functions `projectile-add-dir-local-variable` and `projectile-delete-dir-local-variable` wrap their built-in counterparts. They always use `.dir-locals.el` from the root of the current Projectile project.
+* Add a new defcustom (`projectile-dirconfig-file`) controlling the name of the file used as Projectile’s root marker and configuration file.
 
```

Comment:
Shouln't the new projectile-root-marked function be mentioned in the changelog as well?

Reply:
Yeah, it should. Those things should also be covered somewhere in the docs, so it's easier to discover but I'll do this myself when I can.